cookbooks/bcpc/templates/default/carbon/carbon.conf.erb

```diff
@@ -48,9 +48,9 @@ MIN_RESET_INTERVAL=121
 METRIC_CLIENT_IDLE_TIMEOUT = <%="#{node[:bcpc][:graphite][:carbon][:relay][:idle_timeout]}"%> 
 
 [aggregator]
-LINE_RECEIVER_INTERFACE = <%="#{node[:bcpc][:graphite][:ip]}"%>
+LINE_RECEIVER_INTERFACE = <%="#{node[:bcpc][:management][:ip]}"%>
```

Comment:
carbon_cache is using node[:bcpc][:management][:ip] to receive input from carbon_relay.
My intention is to put carbon_aggregator in front of carbon_cache which mean it should use the ip that carbon_cache is using. I'm not sure how separating out different clusters works, but that probably needs another PR to achieve.